model/host/host.go

```diff
@@ -810,12 +810,16 @@ func (h *Host) SetProvisioning(ctx context.Context) error {
 }
 
 // SetDecommissioned sets the host as decommissioned. If checkTaskGroup is set,
-// we only update the host if it hasn't started running a task group.
-func (h *Host) SetDecommissioned(ctx context.Context, user string, checkTaskGroup bool, logs string) error {
+// we only update the host if it hasn't started running a task group. If terminateIfBusy
+// isn't set, only update the host if there is no running task.
+func (h *Host) SetDecommissioned(ctx context.Context, user string, checkTaskGroup, terminateIfBusy bool, logs string) error {
```

Comment:
nit: I would call this `decommissionIfBusy` (or maybe `checkRunningTask`) since this is decommissioning the host, not terminating it.

Reply:
done